### DISCUSSION TITLE
article/notionnext-deploy-web3.0-4everland

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#72] [Announcements]
# article/notionnext-deploy-web3.0-4everland

部署在Web3.0平台上的网站能够提供更加安全、透明、去中心化和创新的服务，为用户带来更好的体验和价值。NotionNext支持部署在去中心化的Web3.0平台上，这将为用户提供更加安全和可靠的服务，同时也将为网站带来新的商业机会和竞争优势。

https://docs.tangly1024.com/article/notionnext-deploy-web3.0-4everland

**Comment by tiandankanfeng** [comment]
0:35:15.814
build starting...
00:35:27.509
node[15]: ../src/node_platform.cc:68:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.
00:35:27.510
1: 0xb7a940 node::Abort() [node]
00:35:27.510
2: 0xb7a9be [node]
00:35:27.511
3: 0xbe98be [node]
00:35:27.511
4: 0xbe99a1 node::NodePlatform::NodePlatform(int, v8::TracingController*, v8::PageAllocator*) [node]
00:35:27.512
5: 0xb38f5b node::InitializeOncePerProcess(std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, node::ProcessFlags::Flags) [node]
00:35:27.512
6: 0xb395ab node::Start(int, char**) [node]
00:35:27.512
7: 0x7f068702818a [/lib/x86_64-linux-gnu/libc.so.6]
00:35:27.512
8: 0x7f0687028245 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
00:35:27.513
9: 0xabbdee _start [node]
00:35:27.517
Aborted (core dumped)


失败了。。
